src/eos_gates.js

```diff
@@ -63,9 +120,48 @@ const Application = new Lang.Class({
         return this._konami.keyRelease(event);
     },
 
+    getHelpMessage: function() {
+        if (!this.replacement)
+            return _("You can install applications from our %s.").format(link(_("App Store"),
```

Comment:
Also the previous code for this linked to endlessm-app://org.gnome.Software rather than eos-app-store.